client/src/store/modules/scheduledEvent/actions.js

```diff
@@ -35,13 +54,23 @@ export default {
                 endDate
             );
 
-            commit(mutations.CLEAR_SCHEDULED_EVENTS);
-            commit(mutations.SET_SCHEDULED_EVENTS, events);
+            if (page === 1) {
+                commit(mutations.CLEAR_SCHEDULED_EVENTS);
+            }
+
+            commit(mutations.SET_SCHEDULED_EVENTS_PAGINATION, events.meta);
+            commit(mutations.SET_SCHEDULED_EVENTS, events.data);
+            commit('loader/' + loaderMutations.SET_LOADING, false, {
+                root: true
+            });
         } catch (error) {
             commit(
                 SET_ERROR_NOTIFICATION,
                 error?.response?.data?.message || error.message
             );
```

Comment:
If token expired => 401 error => we must redirect user to login page. Use existing action from 'auth' module